docs/get-started/Velox.md

```diff
@@ -153,31 +153,22 @@ mvn clean package -Pbackends-velox -Prss -Pspark-3.3 -DskipTests
 
 ## 2.3 HDFS support
 
-Hadoop hdfs support is ready via the [libhdfs3](https://github.com/apache/hawq/tree/master/depends/libhdfs3) library. The libhdfs3 provides native API for Hadoop I/O without the drawbacks of JNI. It also provides advanced authentatication like Kerberos based. Please note this library has serveral depedencies which may require extra installations on Driver and Worker node.
-On Ubuntu 20.04 the required depedencis are libiberty-dev, libxml2-dev, libkrb5-dev, libgsasl7-dev, libuuid1, uuid-dev. The packages can be installed via below command:
-```
-sudo apt install -y libiberty-dev libxml2-dev libkrb5-dev libgsasl7-dev libuuid1 uuid-dev
-```
-To build Gluten with HDFS support, below command is provided:
```

Comment:
Similarly, the build guide for s3 support is outdated. Could you please update that part also? See https://github.com/oap-project/gluten/blob/main/docs/get-started/Velox.md#25-aws-s3-support
Thanks!

Reply:
@PHILO-HE I will follow-up on this